packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js

```diff
@@ -162,25 +171,36 @@ const WatchlistCard = ({
             />
             <div className="details">
               {<div className="price">{stockData.price || 'N/A'}</div>}
-              {<div className="delta">{stockData.delta || ''}</div>}
-
-              <div className="percentage">
+              <div
+                className={classNames({
+                  'percentage price_positive': stockData.percentage > 0,
+                  'percentage price_negative': stockData.percentage < 0
```

Comment:
Could use a ternary instead? `stockData.percentage > 0 ? 'percentage price_positive' : 'percentage price_negative'
Or classNames cx 'percentage', then the ternary statement for adding price_positive or price_negative

Reply:
removed the classNames function and used a normal ternary instead, makes code a lot more readable.